src/commands/hld/reconcile.ts

```diff
@@ -104,6 +104,13 @@ export interface IReconcileDependencies {
     serviceConfig: IBedrockServiceConfig
   ) => Promise<IExecResult>;
 
+  configureChartForRing: (
```

Comment:
change to `configureChartForRing: typeof configureChartForRing`

Reply:
fixed